eng/testing/RunnerTemplate.sh

```diff
@@ -79,23 +79,19 @@ function invoke_xunitlogchecker {
   local dump_folder=$1
 
   total_dumps=$(find $dump_folder -name "*.dmp" | wc -l)
-  
+
   if [[ $total_dumps > 0 ]]; then
     echo "Total dumps found in $dump_folder: $total_dumps"
-    xunitlogchecker_file_name="$HELIX_CORRELATION_PAYLOAD/XUnitLogChecker.dll"
-    dotnet_file_name="$RUNTIME_PATH/dotnet"
+    xunitlogchecker_file_name="$HELIX_CORRELATION_PAYLOAD/XUnitLogChecker"
 
-    if [[ ! -f $dotnet_file_name ]]; then
-      echo "'$dotnet_file_name' was not found. Unable to run XUnitLogChecker."
-      xunitlogchecker_exit_code=1
-    elif [[ ! -f $xunitlogchecker_file_name ]]; then 
+    if [[ ! -f $xunitlogchecker_file_name ]]; then
       echo "'$xunitlogchecker_file_name' was not found. Unable to print dump file contents."
```

Comment:
nit: Should we match the error message with the one in Windows?:

```suggestion
      echo "XUnitLogChecker does not exist in the expected location: $xunitlogchecker_file_name"
```